tests/e2e/memgraph.py

```diff
@@ -62,15 +62,18 @@ def __init__(self, binary_path=MEMGRAPH_BINARY, use_ssl=False):
         self.binary_path = binary_path
         self.args = None
         self.proc_mg = None
-        self.conn = None
         self.ssl = use_ssl
 
-    def query(self, query):
-        cursor = self.conn.cursor()
-        cursor.execute(query)
-        return cursor.fetchall()
+    def execute_setup_queries(self, setup_queries):
+        conn = mgclient.connect(host=self.host, port=self.bolt_port, sslmode=self.ssl)
+        conn.autocommit = True
+        cursor = conn.cursor()
+        for query in setup_queries:
+            cursor.execute(query)
+        cursor.close()
+        conn.close()
 
-    def start(self, restart=False, args=[]):
+    def start(self, restart=False, args=[], setup_queries=[]):
```

Comment:
pls rewrite this to be either `None` or required argument, since we can get invalid behaviour with initializing lists in Python, same with `args` argument

Reply:
handle args and setup queries in the method